assets/data/cards_pt-br.json

```
@@ -296,6 +296,11 @@
       "title": "Pull Request",
       "description": "Trata-se de uma requisição de alteração dentro de um projeto. Essa requisição pode ser feita para corrigir um problema, adicionar uma nova funcionalidade, sugerir alterações, entre outros motivos. Essa requisição de alteração é feita para que haja uma avaliação do que está sendo alterado dentro do projeto, entendendo de faz sentido para o contexto daquele momento. E essa requisição pode ficar pendente até que pessoas responsáveis ou que tenham a permissão, aprovem. Outras regras podem ser adicionadas nessas requisições, como build da aplicação, rodar testes com sucesso e o que mais fizer sentido para o projeto em questão.",
       "tags": ["Conceito"]
-    }    
+    },
+    {
+      "title": "IDE",
+      "description": "IDE é a abreviação de 'Integrated Development Environment' termo que em português siginifica 'Ambiente de Desenvolvimento Integrado'. É um programa com interfaces gráficas que desenvolvedores utilizam para criar aplicações. Normalmente são compostos por editor de código fonte e ferramentas para depuração de erros. Podem conter também outras ferramentas que auxiliem a criação de testes para tornar o processo de desenvolvimento mais eficiente. Ex.: Android Studio, Eclipse e Visual Code.",
```

A sua explicação ficou boa e você explicou de forma direta! Apesar disso, creio que dá para deixar mais claro ainda.

Vou deixar uma sugestão com algumas correções :)

```suggestion
      "description": "IDE é a abreviação de 'Integrated Development Environment', que em português significa 'Ambiente de Desenvolvimento Integrado'. Trata-se de um programa com interfaces gráficas que facilitam a interação dos desenvolvedores na criação de aplicações. Normalmente um IDE inclui um editor de código fonte e ferramentas de depuração para identificar e corrigir erros. Exemplos populares de IDEs incluem Android Studio, Eclipse e Visual Studio Code.",
```